lib/grpcx/server/interceptors/rescue.rb

```diff
@@ -1,7 +1,11 @@
+require 'grpclb/server'
+
 module Grpcx
-  module Server
+  class Server < Grpclb::Server
```

Comment:
I dislike this, but seems no way round it, except, probably, changing naming scheme to smth like `Grpcx::Interceptors::Server::Rescue`